editor/patches/istextorbinary+5.11.0.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/istextorbinary/package.json b/node_modules/istextorbinary/package.json
+index 8386188..cdfa9fa 100644
+--- a/node_modules/istextorbinary/package.json
++++ b/node_modules/istextorbinary/package.json
+@@ -220,16 +224,6 @@
+       "bevry"
+     ]
+   },
+-  "exports": {
+-    "node": {
+-      "import": "./edition-es2019-esm/index.js",
+-      "default": "./index.cjs",
+-      "require": "./edition-esnext/index.js"
+-    },
+-    "browser": {
+-      "import": "./edition-browsers/index.js"
+-    }
+-  },
```

Comment:
Can this issue be fixed by updating `istextorbinary` to `6.0.0`?

Reply:
I tried it last week on a branch and did not help